src/Sulu/Bundle/WebsiteBundle/Controller/CacheController.php

```diff
@@ -53,13 +55,21 @@ public function __construct(
      *
      * @return JsonResponse
      */
-    public function clearAction()
+    public function clearAction(Request $request)
     {
-        if (!$this->checkLivePermissionForAllWebspaces()) {
+        $webspaceKey = $request->query->get('webspaceKey');
+        if ($webspaceKey && !$this->checkLivePermissionForWebspace($webspaceKey)) {
             return new JsonResponse(null, 403);
+        } elseif (!$this->checkLivePermissionForAllWebspaces()) {
+            return new JsonResponse(null, 403);
+        }
+
+        $tags = [];
+        if ($webspaceKey) {
+            $tags[] = \sprintf('%s-%s', WebspaceReferenceStore::WEBSPACE_REFERENCE_ALIAS, $webspaceKey);
         }
 
-        $this->cacheClearer->clear();
+        $this->cacheClearer->clear($tags);
```

Comment:
its strange to pass an empty array here to clear the cache for all webspaces, isnt it? 
i would pass `null` to clear everything

Reply:
```suggestion
        $this->cacheClearer->clear(\empty($tags) ? null : $tags);
```